### DISCUSSION TITLE
quick hack to sync kitty colors to foot

### DISCUSSION THREAD

**Discussion by JosiasPires** [prasanthrangan/hyprdots#1890] [Show and tell]
The code is very poorly done but it does the job, made for those who use foot instead of kitty and want to integrate the colors with the rest of the system. contributions are welcome. I take this opportunity to ask the developers to create an option to choose foot instead of kitty.

https://github.com/JosiasPires/paw

**Comment by kRHYME7** [comment]
Is it possible to have a separate file for theming foot?.

**Reply by JosiasPires** [reply]
actually yes, you can do that by writing include=/path/to/file at the start of 'foot.ini' and create a separate file for theming, this way it should be much easier 😅

**Reply by JosiasPires** [reply]
i'll improve the code now that I know that, thanks!

**Reply by kRHYME7** [reply]
Cool now we can derive a template from that file.
